indico/modules/events/abstracts/blueprint.py

```diff
@@ -119,6 +119,8 @@
                      RHReviewAbstractForTrack, methods=('POST',), defaults=defaults)
     _bp.add_url_rule(prefix + '/<int:abstract_id>/judge', 'judge_abstract',
                      RHJudgeAbstract, methods=('POST',), defaults=defaults)
+    _bp.add_url_rule(prefix + '/<int:abstract_id>/review/tracks', 'edit_review_tracks',
```

Comment:
I thought that since it's related with the **reviewed**_for_tracks, it could use the `/review/` path.